### ISSUE TITLE
What is the argument --pretrained pretrained.pth.tar?

### ISSUE THREAD

**Issue by cuong-pv** [ZiweiWangTHU/GMPQ#4]
Thanks for your exciting work. When running your code, I am not sure the input "pretrained.pth.tar" in the main.py when fine-tuning on imagenet. It seems the weight of pretrained full precision of Resnet18 on imagenet. Is the is right?.

**Comment by Euphoria16** [comment]
Yes. We finetune the quantized model starting from a pre-trained model of full precision.

**Comment by Euphoria16** [comment]
> Thanks for your exciting work. When running your code, I am not sure the input "pretrained.pth.tar" in the main.py when fine-tuning on imagenet. It seems the weight of pretrained full precision of Resnet18 on imagenet. Is the is right?.

That's right. The quantized model is usually fine-tuned starting from a pre-trained model of full precision.
